src/index.html

```diff
@@ -118,7 +118,7 @@
             <button type="button" class="btn btn-primary" data-bind="click: send">Send</button>
           </div>
           <div class="btn-group">
-            <button type="button" class="btn btn-primary" aria-haspopup="true" aria-expanded="false" data-bind="click: saveBookmarkDialog">Save</button>
+            <button type="button" id="save-button" class="btn btn-primary" aria-haspopup="true" aria-expanded="false" data-bind="click: saveBookmarkDialog" disabled="true">Save</button>
```

Comment:
to enable or disable button you can use the `enable` binding into `data-bind` attribute.

More info how to use it [here](https://knockoutjs.com/documentation/enable-binding.html)